packages/react-components/react-infobutton/src/components/InfoButton/useInfoButton.tsx

```diff
@@ -56,7 +57,7 @@ export const useInfoButton_unstable = (props: InfoButtonProps, ref: React.Ref<HT
     content: resolveShorthand(props.content, {
       required: true,
       defaultProps: {
-        role: 'dialog',
+        role: 'note',
```

Comment:
There's [not a lot of information about `note`](https://w3c.github.io/aria/#note) so it's not clear to me this is the correct role here. Can you provide some details on how you settled on this role?

I would have expected [`tooltip`](https://w3c.github.io/aria/#tooltip) here but that may not be the correct thing to do.

Reply:
This was suggested by Sarah, the main points for choosing `note` over `tooltip` are:
* `tooltip`'s main difference with note is that it has a kind of live region side-effect that (screen readers treat is as sort of live region) we don't really want since we are moving focus to it.
* In theory, `tooltip` is not supposed to contain interactive items and InfoButton will have if needed. Note is more flexible with this aspect.
* `note` means something related to its trigger, similar to a footnote.

Reply:
Thanks for the details!